streamdeck_ui/gui.py

```diff
@@ -625,16 +631,33 @@ def enable_button_configuration(ui: Ui_ButtonForm, enabled: bool):
     ui.write.setVisible(pynput_supported)
 
 
-def prepare_button_state_form_text_font_list(ui: Ui_ButtonForm) -> None:
+def prepare_button_state_form_text_font_list(ui: Ui_ButtonForm, current_font_family: str) -> None:
     """Prepares the font selection combo box with all available fonts"""
     blocker = QSignalBlocker(ui.text_font)
     try:
         ui.text_font.clear()
-        font_files = [f for f in os.listdir(os.path.join(FONTS_PATH)) if f.endswith(".ttf")]
+        ui.text_font.clearEditText()
+        for i, font_family in enumerate(FONTS_DICT):
+            ui.text_font.addItem(font_family)
+            font = QFont(font_family)
+            ui.text_font.setItemData(i, font)
```

Comment:
ui.text_font.setItemData(i, font, Qt.FontRole)  # type: ignore [attr-defined]

Reply:
it should be fixed now. please if you can confirm that it's fixed just mark as resolve the conversation. thank you in advance